src/pages/DashboardAdmin/components/PersonnelDoctorDetail.tsx

```diff
@@ -0,0 +1,315 @@
+import { useEffect, useState } from "react";
+import { useForm } from "react-hook-form";
+import { getParticularUser, updateParticularDoctor } from "../../../api/users";
+import { getSpecialtyByHospital } from "../../../api/utilities";
+import { IDoctor } from "../../../models/IDoctor";
+import { BackIcon } from "../styles/AddPersonnel.style";
+import ArrowLeft from "../../../resources/ArrowLeft.svg";
+import { useHistory, useParams } from "react-router";
+import { ConfirmationModal } from "../../../components/ConfirmationModal";
+
+type SelectSpecialty = {
+  label: string;
+  value: number;
+};
+
+type doctorProps = {
+  id: string;
+};
```

Comment:
[Not a blocker]

Realmente si este id es el id de la url, te recomendaria llamarle `doctorParams`, mas bien por si en un futuro te interesa tambien agregarle props a ese componente que ese otro no te lo impida